frontend/src/libs/import-export/parse-spreadsheet.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-env node */
+import XLSX from "xlsx";
+
+export function parseSpreadsheet(fileName) {
+    const workbook = XLSX.readFile(
+        __dirname + `/../../tests/import-export-data/${fileName}`
```

Comment:
What is this? This code is not appropriate here. You should only be reading files from the file system within a test, but this function is exported to the entire app...